public/static/themes/simple.css

```
@@ -1,8 +1,42 @@
 /* @import url('./catppuccin-mocha.css'); */
 @font-face {
   font-family: Rubik;
-  src: url('https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600;700;800&display=swap');
-  fallback: sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  font-display: swap;
+  src: url("https://fonts.gstatic.com/s/rubik/v28/iJWKBXyIfDnIV7nBrXw.woff2") format('woff2');
+}
+
+@font-face {
+  font-family: Rubik;
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url("https://fonts.gstatic.com/s/rubik/v28/iJWKBXyIfDnIV7nBrXw.woff2") format('woff2');
+}
+
+@font-face {
+  font-family: Rubik;
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: url("https://fonts.gstatic.com/s/rubik/v28/iJWKBXyIfDnIV7nBrXw.woff2") format('woff2');
+}
+
+@font-face {
+  font-family: Rubik;
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("https://fonts.gstatic.com/s/rubik/v28/iJWKBXyIfDnIV7nBrXw.woff2") format('woff2');
+}
+
+@font-face {
+  font-family: Rubik;
+  font-style: normal;
+  font-weight: 600;
+  font-display: swap;
```

```suggestion
@font-face {
    font-family: 'Rubik';
    font-style: normal;
    font-weight: 200 600;
    font-stretch: 0% 200%;
    font-display: swap;
    src: url(https://fonts.gstatic.com/s/rubik/v28/iJWKBXyIfDnIV7nErXyi0A.woff2) format('woff2');
    optional: sans-serif;
}
```

I think I would suggest instead of having separate font-faces for the same font. You can just provide one single font face as the font family you are using seems to be a variable font so you can effectively write it in one single `@font-face` attribute. I hope you got my point :slightly_smiling_face: 

Also, for more information on this, I would recommend the following resources to go through:
- https://stackoverflow.com/questions/28279989/multiple-font-weights-one-font-face-query

> [!Note]
> In the above link see the answer/reply with the title **Update 2022: use variable fonts**. 

- https://fonts.google.com/variablefonts?vfquery=rubik#axis-definitions

---

Looks good, except for the the `optional` property, which is not passing CodeFactor nor is my css linter recommending that I push it. So does the single quotation marks around the font-family name, which is apparently unnecessary.

---

> Looks good, except for the the `optional` property, which is not passing CodeFactor nor is my css linter recommending that I push it. So does the single quotation marks around the font-family name, which is apparently unnecessary.

Yes, they are not really required because I tested it on my side. It seems to work without quotation marks as well. So I would suggest removing it and also try running the app on your side too. :slightly_smiling_face:

---

Tested on my end just now, everything works.